lib/tilebag.rb

```diff
@@ -0,0 +1,26 @@
+module Scrabble
+  class TileBag
+    attr_reader :tile_bag
+
+    def initialize
+      @tile_bag = [ "A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B", "C",
```

Comment:
While impressive-looking, a big literal array like this is hard to manage. Say your boss comes in with the news that Scrabble should now include 9 "E"s - how many do you have now? You'll have to count, which is error prone.

A better way of storing this information might be something like you did in the `Scoring` class, a hash where each letter maps to its number of tiles. You'd then want to iterate over it to generate the tile bag itself.